kalite/i18n/api_views.py

```diff
@@ -3,34 +3,52 @@
 from django.conf import settings; logging = settings.LOG
 from django.utils.translation import gettext as _
 from django.views.decorators.csrf import csrf_exempt
+from django.shortcuts import redirect
 
 from .base import get_default_language, set_default_language, set_request_language
 from fle_utils.internet.classes import JsonResponse, JsonResponseMessageError
 from fle_utils.internet.decorators import api_handle_error_with_json
 
-
 @csrf_exempt
 @api_handle_error_with_json
 def set_server_or_user_default_language(request):
-    if request.method == 'GET':
-        return JsonResponseMessageError(_("Can only handle default language changes through POST requests"), status=405)
 
+    returnUrl = ''
+    allUsers = ''
+
+    # by allowing GET requests we can use this script to switch
+    # languages on the fly with a link like this:
+    #   http://my.host:8008/api/i18n/set_default_language/?lang=en&returnUrl=/learn/khan/math
+    # That would change the language for the current user. You
+    # can also switch the default language for all users with
+    # the addition of '&allUsers=1' argument
+    if request.method == 'GET':
+        data = request.GET
+        if 'returnUrl' in data:
+            returnUrl = data['returnUrl']
```

Comment:
This is a small thing, you don't need to bother - but in Python, camelCase is almost entirely never used for variable names :) But nevermind in this case :)